client/src/components/PageLayout/Header.tsx

```diff
@@ -30,6 +30,7 @@ const HeaderItem = forwardRef<HTMLDivElement, Props>((props, ref) => {
       {...props}
       w="full"
       as="header"
+      height={'90px'}
```

Comment:
The increased spaces seems off-putting to me in small screens. what about `4.5em`, so it looks like this 

![image](https://user-images.githubusercontent.com/88248797/200184603-bb2a6688-f415-4aba-a0df-8660176d07e2.png)


```suggestion
      height={'4.5em'}
```